signup.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" as="style" crossorigin
+    href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <link rel="stylesheet" href="singup.css">
+  <title>signup</title>
+
+</head>
+
+<body>
+  <main>
+    <a href="index.html" class="logo">
+      <img src="img/logo.png">
+    </a>
+    <div id="main-body">
+      <div id="singup-container">
+        <div id="input-group">
+
+          <div id="email-body">
+            <p class="bold">이메일</p>
+            <input id="member-email" name="email" class="member-input" placeholder="이메일을 입력해주세요." type="email">
+            <p id="email-error" style="color: red;"></p>
+          </div>
+
+          <div id="nickname-body">
+            <p class="bold">닉네임</p>
+            <input id="member-nickname" name="nickname" class="member-input" placeholder="닉네임을 입력해주세요.">
+          </div>
+
+          <div id="password-body">
+            <p class="bold">비밀번호</p>
+            <div id="password-container" class="input-container">
+              <input id="member-password" name="pw" class="member-input1" placeholder="비밀번호를 입력해주세요." type="password">
+              <img id="togglePassword" src="img/eye_button.png" alt="eye-icon" class="eye-icon">
+            </div>
+            <p id="password-error" style="color: red;"></p>
+          </div>
+
+          <div id="checkpass-body">
+            <p class="bold">비밀번호 확인</p>
+            <div id="chkpassword-container" class="input-container">
+              <input id="member-chkpassword" name="chekpw" class="member-input1" placeholder="비밀번호를 다시 한 번 입력해주세요." type="password">
+              <img id="toggleChkPassword" src="img/eye_button.png" alt="eye-icon" class="eye-icon">
+            </div>
+            <p id="chkpassword-error" style="color: red;"></p>
+          </div>
+
+        </div>
+        <button id="singup-btn" class="singup-btn" disabled>회원가입</button>

```

Comment:
signup 오타 있네요 (js,css 파일쪽도 바꿔야합니다)
```suggestion
        <button id="signup-btn" class="signup-btn" disabled>회원가입</button>

```